beta-src/src/App.tsx

```diff
@@ -2,8 +2,14 @@ import * as React from "react";
 import "./assets/css/App.css";
 import Box from "@mui/material/Box";
 import WDMain from "./components/ui/WDMain";
+import { useAppDispatch } from "./state/hooks";
+import { fetchGameOverview } from "./state/game/game-api-slice";
 
 const App: React.FC = function (): React.ReactElement {
+  const urlParams = new URLSearchParams(window.location.search);
```

Comment:
hmm, jin is doing this in one of his diffs. did you mean to commit this or was this a test?

Reply:
I'm committing it, and I'm coordinating with him so that he pulls this in since we want this to be fetched in a high level component like App, this way the other components can just use the already fetched data.

Reply:
should we store the id in state?